### ISSUE TITLE
error in rendering pdf documents due to using tbl-cap

### ISSUE THREAD

**Issue by abduazizR** [quarto-dev/quarto-cli#9979]
### Bug description

Hi 

I was trying to create a PDF document in Quarto while using tbl-cap. The problem is that the document does not compile and gives me this error message

```
ERROR: 
compilation failed- error
Package ltcaption Error: \caption is not available in longtable* environment.

See the ltcaption package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.263 \caption{hey hello}
                         \tabularnewline 

see test_pdf_latex.log for more information.
```

I used to have no issues with this before. 

### Steps to reproduce

````

---
title: "Untitled"
format: pdf
editor: visual
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

## Running Code

When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:

```{r}
1 + 1
```

You can add options to executable code like this

```{r}
#| echo: false
2 * 2
```

The `echo: false` option disables the printing of code (only output is displayed).

```{r}
#| tbl-cap: hey hello
pacman::p_load(flextable)
iris |> head() |> flextable()
```


````

### Expected behavior

Printing PDF document with table captions

### Actual behavior

```
ERROR: 
compilation failed- error
Package ltcaption Error: \caption is not available in longtable* environment.

See the ltcaption package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.263 \caption{hey hello}
                         \tabularnewline 

see test_pdf_latex.log for more information.
```

### Your environment

```
R version 4.4.0 (2024-04-24 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 22631)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
 [1] flextable_0.9.6 janitor_2.2.0   duckdb_0.10.2   DBI_1.2.2      
 [5] arrow_16.1.0    here_1.0.1      nhanesA_1.1     lubridate_1.9.3
 [9] forcats_1.0.0   stringr_1.5.1   dplyr_1.1.4     purrr_1.0.2    
[13] readr_2.1.5     tidyr_1.3.1     tibble_3.2.1    ggplot2_3.5.1  
[17] tidyverse_2.0.0

loaded via a namespace (and not attached):
 [1] gtable_0.3.5            xfun_0.44               tzdb_0.4.0             
 [4] vctrs_0.6.5             tools_4.4.0             generics_0.1.3         
 [7] curl_5.2.1              fansi_1.0.6             pacman_0.5.1           
[10] pkgconfig_2.0.3         data.table_1.15.4       uuid_1.2-0             
[13] assertthat_0.2.1        lifecycle_1.0.4         compiler_4.4.0         
[16] textshaping_0.4.0       munsell_0.5.1           snakecase_0.11.1       
[19] httpuv_1.6.15           fontquiver_0.2.1        fontLiberation_0.1.0   
[22] htmltools_0.5.8.1       yaml_2.3.8              later_1.3.2            
[25] pillar_1.9.0            crayon_1.5.2            gfonts_0.2.0           
[28] openssl_2.2.0           mime_0.12               fontBitstreamVera_0.1.1
[31] zip_2.3.1               digest_0.6.35           tidyselect_1.2.1       
[34] rvest_1.0.4             stringi_1.8.4           fastmap_1.2.0          
[37] rprojroot_2.0.4         grid_4.4.0              colorspace_2.1-0       
[40] cli_3.6.2               magrittr_2.0.3          crul_1.4.2             
[43] utf8_1.2.4              foreign_0.8-86          withr_3.0.0            
[46] promises_1.3.0          gdtools_0.3.7           scales_1.3.0           
[49] bit64_4.0.5             timechange_0.3.0        rmarkdown_2.27         
[52] officer_0.6.6           httr_1.4.7              bit_4.0.5              
[55] ragg_1.3.2              askpass_1.2.0           hms_1.1.3              
[58] evaluate_0.23           shiny_1.8.1.1           knitr_1.46             
[61] rlang_1.1.3             Rcpp_1.0.12             xtable_1.8-4           
[64] glue_1.7.0              httpcode_0.3.0          selectr_0.4-2          
[67] xml2_1.3.6              renv_1.0.7              jsonlite_1.8.8         
[70] rstudioapi_0.16.0       R6_2.5.1                plyr_1.8.9             
[73] systemfonts_1.1.0    
```

### Quarto check output

```
Quarto 1.5.43
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.5.43
      Path: C:\Users\aabdel51\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\Users\aabdel51\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2024

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.9.13 (Conda)
      Path: C:/ProgramData/Anaconda3/python.exe
      Jupyter: 4.11.1
      Kernels: python3

(|) Checking Jupyter engine render....Traceback (most recent call last):
  File "C:\Users\aabdel51\AppData\Local\Programs\Quarto\share\jupyter\jupyter.py", line 21, in <module>
    from notebook import notebook_execute, RestartKernel
  File "C:\Users\aabdel51\AppData\Local\Programs\Quarto\share\jupyter\notebook.py", line 19,
in <module>
(/) Checking Jupyter engine render....    from nbclient import NotebookClient
  File "C:\ProgramData\Anaconda3\lib\site-packages\nbclient\__init__.py", line 6, in <module>    from .client import NotebookClient, execute  # noqa: F401
  File "C:\ProgramData\Anaconda3\lib\site-packages\nbclient\client.py", line 14, in <module>
    from jupyter_client import KernelManager
  File "C:\ProgramData\Anaconda3\lib\site-packages\jupyter_client\__init__.py", line 8, in <module>
    from .asynchronous import AsyncKernelClient  # noqa
  File "C:\ProgramData\Anaconda3\lib\site-packages\jupyter_client\asynchronous\__init__.py",
line 1, in <module>
    from .client import AsyncKernelClient  # noqa
  File "C:\ProgramData\Anaconda3\lib\site-packages\jupyter_client\asynchronous\client.py", line 6, in <module>
    from jupyter_client.channels import HBChannel
  File "C:\ProgramData\Anaconda3\lib\site-packages\jupyter_client\channels.py", line 12, in <module>
    import zmq.asyncio
  File "C:\ProgramData\Anaconda3\lib\site-packages\zmq\__init__.py", line 103, in <module>
    from zmq import backend
  File "C:\ProgramData\Anaconda3\lib\site-packages\zmq\backend\__init__.py", line 31, in <module>
    raise original_error from None
  File "C:\ProgramData\Anaconda3\lib\site-packages\zmq\backend\__init__.py", line 26, in <module>
    _ns = select_backend(first)
  File "C:\ProgramData\Anaconda3\lib\site-packages\zmq\backend\select.py", line 31, in select_backend
    mod = import_module(name)
  File "C:\ProgramData\Anaconda3\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "C:\ProgramData\Anaconda3\lib\site-packages\zmq\backend\cython\__init__.py", line 6, in <module>
    from . import (
ImportError: DLL load failed while importing _device: The specified module could not be found.
[>] Checking Jupyter engine render....OK
```

**Comment by cscheid** [comment]
I can't repro this with the latest version of flextable from CRAN, 0.9.6.

**Comment by abduazizR** [comment]
The error occurs with me even without using Flextable. I had the same message when I tried printing `gt` tables

**Comment by cscheid** [comment]
This was fixed in between 1.5.43 and 1.5.44 (I see a failure in .43, and not in .44). I'm going to go ahead and close this one.
